src/main/python/deep_qa/data/instances/question_passage_instance.py

```diff
@@ -99,7 +99,8 @@ def pad(self, max_lengths: Dict[str, int]):
         max_lengths_tmp['word_sequence_length'] = max_lengths_tmp['num_question_words']
         self.question_indices = self.pad_word_sequence(self.question_indices, max_lengths_tmp)
         max_lengths_tmp['word_sequence_length'] = max_lengths_tmp['num_passage_words']
-        self.passage_indices = self.pad_word_sequence(self.passage_indices, max_lengths_tmp)
+        self.passage_indices = self.pad_word_sequence(self.passage_indices, max_lengths_tmp,
```

Comment:
Yeah, I was going to fix those tests today, but had too many meetings.  I'll do this soon.  And I think this is the right decision for all of those models - if you're truncating the passage, you should keep the beginning of the passage (for WDW, e.g., the correct entity is biased towards appearing early in the document).